content/rocm.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Install ROCm
+description: >
+  Instructions for setting up ROCm for HIP & OpenCL workloads on AMD GPUs
+keywords:
+  - AMD
+  - ROCm
+  - ROCr
+  - ROCt
+  - ROCk
+  - Radeon Open Compute
+  - HIP
+  - OpenCL
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: software
+tableOfContents: true
+---
+
+AMD GPUs use [HIP](https://rocm.docs.amd.com/projects/HIP/en/latest/) (Heterogeneous-Compute Interface for Portability) and [OpenCL](https://www.khronos.org/opencl/) (Open Computing Language) to run compute workloads, similar to CUDA on NVIDIA GPUs. In order to use HIP and OpenCL, [ROCm](https://www.amd.com/en/products/software/rocm.html) (the Radeon Open Compute modules) must be installed. The ROCm package interfaces with the AMDGPU driver built into the default Pop!\_OS kernel, and does not require installing any DKMS packages.
+
+The below instructions correspond with the necessary steps from [AMD's installation documentation for Ubuntu](https://rocm.docs.amd.com/projects/install-on-linux/en/latest/how-to/native-install/ubuntu.html).
+
+## Installation on Pop!\_OS 22.04
+
+First, add AMD's official ROCm repository to the system and set the appropriate apt pin priority using the below commands:
+
+```
+wget https://repo.radeon.com/rocm/rocm.gpg.key -O - | gpg --dearmor | sudo tee /etc/apt/keyrings/rocm.gpg > /dev/null
+echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/rocm.gpg] https://repo.radeon.com/rocm/apt/6.1 jammy main" | sudo tee --append /etc/apt/sources.list.d/rocm.list
+echo -e 'Package: *\nPin: release o=repo.radeon.com\nPin-Priority: 600' | sudo tee /etc/apt/preferences.d/rocm-pin-600
+```
+
+Next, update the package lists and install the `rocm` package:
+
+```
+sudo apt update
+sudo apt install rocm
+```
+
+Finally, add the current user to the `render` group so the user has permission to run compute workloads:
+
+```
+sudo usermod -a -G render $USER
```

Comment:
What application do you have that requires the `video` group? I did see that in AMD's instructions, but it didn't make sense to me since that group is for things like webcams.

Reply:
See documentation about system groups here: https://wiki.debian.org/SystemGroups 
It looks like GPUs used to be lumped into the `video` group, but systemd added a `render` group to use instead:
- https://github.com/RadeonOpenCompute/ROCm_Documentation/issues/39 
- https://github.com/NixOS/nixpkgs/issues/45071 

Again, if you know of a specific app or use case that actually requires being part of the `video` group, please let me know, and we will certainly add it to the instructions. As of right now, though, it seems like that's simply an outdated instruction for older distributions/versions, and given that this documentation is for Pop!_OS, I don't think we need to have users add themselves to a now-unrelated group unnecessarily.